ocaml/xapi/helpers.ml

```diff
@@ -1260,7 +1260,9 @@ end = struct
     let rewrite_xapi_ssl_config_file ~accept =
       let conf_contents = String.concat "\n" [
           "; autogenerated by helpers.ml"
-        ; "fips = no"
+        ; (match Sys.getenv_opt "CC_PREPARATIONS" with
+          | Some "true"   -> "fips = yes"
```

Comment:
You could say `"fips = " ^ (if fips then "yes" else "no")`, and set `fips` based on the env var as you did, just in case we need to do anything else to turn fips on /off